### PR TITLE
Update backups page to mention our self-managed backups

### PR DESCRIPTION
**Pull request by ebenezerdon**
## What does this PR do?

Updates the backups page in the docs to mention our self-managed backups.

## Test Plan

- Run the app and navigate to `/docs/advanced/security/backups`
- Confirm that the content and link reference are displayed correctly

### CODE REVIEW

**Review by tessamero — COMMENTED** [review]
change "data bases" to "databases"

**Review by tessamero — APPROVED** [review]

File: src/routes/docs/advanced/security/backups/+page.markdoc [hunk 1]
```diff
@@ -10,5 +10,10 @@ Backups are crucial for:
 - Recovering in case of malicious alteration or deletion of data.
 - Reducing downtime in the event of accidental data loss, corruption, or malicious attacks.
 
-Appwrite Cloud uses multiple databases for its products. Some data bases will have access to point-in-time recovery from the past 7 days.
-Other databases performs regular data backups every 4 hours. Each backup is kept for 7 days.
\ No newline at end of file
+Appwrite Cloud provides both automated and self-managed backups.
```
Comment (stnguyen90) [inline src/routes/docs/advanced/security/backups/+page.markdoc:13]: Let's clarify there's:

1. self-managed backups for Appwrite project Databases.
2. backups of the underlying database for disaster recovery.

The order of the topics here should also match the order of the topics below.

File: src/routes/docs/advanced/security/backups/+page.markdoc [hunk 2]
```diff
@@ -10,5 +10,10 @@ Backups are crucial for:
...
+Users can configure automatic backup policies or initiate manual backups directly from the Console.
```
Comment (stnguyen90) [inline src/routes/docs/advanced/security/backups/+page.markdoc:14]: Let's clarify these self-managed backups are for pro plans and above and link to the backups documentation here.

File: src/routes/docs/advanced/security/backups/+page.markdoc [hunk 3]
```diff
@@ -10,5 +10,10 @@ Backups are crucial for:
...
+
+Additionally, some databases support point-in-time recovery
```
Comment (stnguyen90) [inline src/routes/docs/advanced/security/backups/+page.markdoc:16]: Let's clarify these are the internal backups for disaster recovery

**Review by stnguyen90 — CHANGES_REQUESTED** [review]

**Comment by ebenezerdon** [comment]
@stnguyen90 I've pushed a new commit with revision

**Review by stnguyen90 — APPROVED** [review]

File: src/routes/docs/advanced/security/backups/+page.markdoc [hunk 4]
```diff
@@ -1,14 +1,17 @@
...
-Appwrite Cloud uses multiple databases for its products. Some data bases will have access to point-in-time recovery from the past 7 days.
-Other databases performs regular data backups every 4 hours. Each backup is kept for 7 days.
\ No newline at end of file
+Self-managed backups are available for Pro plans and above. These backups allow you to:
+- Configure automatic backup policies
+- Initiate manual backups through the Console
+- Recover from accidental data deletion
+- Restore data to a previous point in time
+
+For detailed information about self-managed backup features, configuration options, and restoration procedures, visit our [Backup Documentation](/docs/products/cloud/backups).
```
Comment (stnguyen90) [inline src/routes/docs/advanced/security/backups/+page.markdoc:15]: This link isn't valid

**Review by stnguyen90 — CHANGES_REQUESTED** [review]

**Review by stnguyen90 — APPROVED** [review]

**Pull request merged**